src/st/lib_psp/st_init_lib.c

```diff
@@ -1,7 +1,30 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 #include "../lib/lib.h"
 
-INCLUDE_ASM("st/lib_psp/psp/lib_psp/1B7F0", func_psp_09253D58);
+#include "../pfn_entity_update.h"
+
+// entity table to allocate new entities
+extern PfnEntityUpdate* PfnEntityUpdates;
+
+extern LayoutEntity* D_8D2E160;
+extern LayoutEntity* D_8D2E234;
+
+// entity table to allocate new entities
+extern PfnEntityUpdate OVL_EXPORT(EntityUpdates)[];
+extern Overlay OVL_EXPORT(Overlay);
+
+extern char D_psp_09276330[];
+extern char D_psp_09276520[];
+extern char D_psp_092766E8[];
+extern char D_psp_092768E0[];
+extern char D_psp_09276AC8[];
```

Comment:
Have held off importing this data for now, since it's related to the cutscene between Alucard and Libarian, and I think it likely has some state machine / control characters baked in to control switching between who's talking and what-not.

It will probably make more sense once `LIB_EntityCutscene` is decompiled

![image](https://github.com/user-attachments/assets/09f9b008-3fd3-408d-bcd8-90e5292c4291)

Reply:
Did you try specifying the offset of the cutscene at `config/assets.us.yaml`? It should decompile the state machine for you

Reply:
This is for PSP side, I don't think we support that for the asset extraction tool do we?

Reply:
The tool should be platform-agnostic. So as long as we create a new asset file for PSP it _should_ work. I am not sure if you want to do it on a separate PR. Maybe it's a good idea?

Reply:
I will try it on a separate PR if it works